.pfnci/schema.yaml

```diff
@@ -248,7 +248,7 @@ cusparselt:
       "12.6":  # TODO(kmaehashi): Tentatively use v0.2.0 for CUDA 12
       "12.8":  # TODO(kmaehashi): Tentatively use v0.2.0 for CUDA 12
       "12.9":  # TODO(kmaehashi): Tentatively use v0.2.0 for CUDA 12
-      "13.0":  # TODO(kmaehashi): Tentatively use v0.2.0 for CUDA 13
+      # Disable cuSPARSELt for CUDA 13 in CuPy v13
```

Comment:
Are there user facing docs that need updating to inform this?

Reply:
Currently use of cuSPARSELt in CuPy is totally hidden from users. We only have an cusparselt python binding wrapper which is undocumented: https://github.com/cupy/cupy/blob/main/cupy_backends/cuda/libs/cusparselt.pyx.